src/Error/FormattedError.php

```diff
@@ -39,6 +39,13 @@
  * This class is used for [default error formatting](error-handling.md).
  * It converts PHP exceptions to [spec-compliant errors](https://facebook.github.io/graphql/#sec-Errors)
  * and provides tools for error debugging.
+ *
+ * @phpstan-type FormattedErrorArray array{
```

Comment:
why is it `FormattedErrorArray` and not just `FormattedError`?

Reply:
Because `FormattedError` is the name of the class we are in.